### PR TITLE
Updating Rule: adding `not benign` check

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

**Comment by morriscode** [comment]
changed tranco check to root_domain as it should've been in the first place.

**Comment by jkamdjou** [comment]
> changed tranco check to root_domain as it should've been in the first place. 

> changed tranco check to root_domain as it should've been in the first place. 

are you sure about this? this will mean that eg free file hosts and free subdomains will come up in Tranco because their root domain is high reputation

File: detection-rules/link_microsoft_low_reputation.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ source: |
   // suspicious link
   and any(body.links,
           (
-            .href_url.domain.domain not in $tranco_1m
+            .href_url.domain.root_domain not in $tranco_1m
```
[the remark this answers is not in the document]
Comment (morriscode) [reply detection-rules/link_microsoft_low_reputation.yml:10]: ??
Tranco is primarily focused on ranking root domains and not individual subdomains or FQDNs.

[the remark this answers is not in the document]
Comment (aidenmitchell) [reply detection-rules/link_microsoft_low_reputation.yml:10]: FQDN? The Tranco list is only root domains, we can't compare `a.b.example.com` to Tranco.

[the remark this answers is not in the document]
Comment (morriscode) [reply detection-rules/link_microsoft_low_reputation.yml:10]: Just now seeing @jkamdjou 's other comment, i still think this is fine, even if the root domain is in tranco the condition is an OR not an AND?

[the remark this answers is not in the document]
Comment (jkamdjou) [reply detection-rules/link_microsoft_low_reputation.yml:10]: you're right 👍 i was mixing up with the alexa list

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**